style.css

```diff
@@ -13,6 +13,7 @@ This theme, like WordPress, is licensed under the GPL.
 
 @import url(https://unpkg.com/@creativecommons/fonts@2020.9.3/css/fonts.css);
 @import url(https://unpkg.com/@creativecommons/vocabulary@2020.11.3/css/vocabulary.css);
+@import url(https://unpkg.com/browse/bulma@0.9.3/);
```

Comment:
I believe the Bulma import should come above the Vocabulary import, so the Vocabulary rules take precedent.